tests/js/tests.js

```diff
@@ -27,12 +27,38 @@ test.describe("top nav bar", () => {
     await expect(pageToC).toHaveScreenshot();
   });
 
+  test("uses custom page ToC icon on tablet dark mode cycle", async ({ page }) => {
+    await setTablet(page);
+    await page.goto("sphinx_guide/lists.html");
+    const pageToC = page.locator("div.content-icon-container");
+    await expect(pageToC).toHaveScreenshot();
+    await click(page, "div.theme-toggle-content button");
```

Comment:
It'd be good to make this a helper function like `toggleColorTheme`. it's less about saving code and more making the code declarative / easier to read